### DISCUSSION TITLE
Add headers to response

### DISCUSSION THREAD

**Discussion by SuPythony** [mirumee/ariadne#652] [Q&A] [answered]
I want to add headers to the response like Set-Cookie for setting a cookie.

Is there a way to do so? If not is there a way to set a cooke in ariadne.

I am using React as frontend.

Please help. Thanks!

**Comment by rafalp** [comment, marked as answer]
Setting cookies on response is murky water when it comes to GraphQL because it doesn't really operate within the request/response context.

Most idiomatic way to achieve this is to return value to be set in cookie as part of mutation's result, and set the cookie in the client, via JS.

**Reply by SuPythony** [reply]
Ok, thanks!
